tests/test_homepage.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.home import HomePage
+
+
+class TestHomePage():
+
+    @pytest.mark.nondestructive
+    def test_home_page(self, base_url, selenium):
```

Comment:
This test is very good. You'll just need to change the code to match the changes I've suggested you make above. Also, you've left out step 2 of issue #39 which says "Verify that the sign in button is visible.". Please add that to the test as well.